onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -239,17 +239,28 @@ def aten_conv_depthwise3d(
     raise NotImplementedError()
 
 
+@torch_op("aten::cross_entropy_loss", trace_only=True)
 def aten_cross_entropy_loss(
-    self: TensorType,
-    target: TensorType,
-    weight: Optional[TensorType] = None,
-    reduction: int = 1,
-    ignore_index: INT64 = -100,
-    label_smoothing: float = 0.0,
-) -> TensorType:
-    """cross_entropy_loss(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean, SymInt ignore_index=-100, float label_smoothing=0.0) -> Tensor"""
-
-    raise NotImplementedError()
+    self: TFloatOrBFloat16,
+    target: Sequence[int],
+    weight: Optional[TFloatOrBFloat16] = None,
+    reduction: int = 1,  # default is 'mean'
+    ignore_index: int = -100,
+    label_smoothing: float = 0.0,  # this was ignored due to ONNX not support
+) -> TFloatOrBFloat16:
+    # cross_entropy_loss(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean, SymInt ignore_index=-100, float label_smoothing=0.0) -> Tensor
+
+    reduction_vals = ["none", "mean", "sum"]
```

Comment:
Can we write some of branches and condition on the int value instead?

Reply:
So that the function is not trace only, even though the logic may become more complex